scipy/stats/_multivariate.py

```diff
@@ -3320,6 +3297,13 @@ class special_ortho_group_gen(multi_rv_generic):
     This generates one random matrix from SO(3). It is orthogonal and
     has a determinant of 1.
 
+    Alternatively, the object may be called (as a function) to fix the `dim`
+    parameter, returning a "frozen" special_ortho_group random variable:
+
+    >>> rv = special_ortho_group(5)
```

Comment:
`ortho_group` is not callable, so it doesn't need any parameters defined at the top level.

Reply:
Done.

Reply:
K. These still say something like "The dim keyword specifies the dimension N." in the description. I guess for now the user can interpret this as being the keyword of the methods. If we get methods rendering, we can fix this then.